server/server.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const app = express();
+const server = require('http').createServer(app);
+const port = 4000;
+
+server.listen(port, function () {
+  console.log(`App listening at http://localhost:${port}`);
+});
+
+app.use(express.static('integration_build'));
```

Comment:
Do we already have a clue what's the difference between integration_build and the "other"?

Reply:
Unfortunately not yet, but I can discuss this next week with @doublethug and add a documentation.